documentation/modules/con-common-configuration-properties.adoc

```diff
@@ -294,18 +294,46 @@ livenessProbe:
 For more information about the `livenessProbe` and `readinessProbe` options, see xref:type-Probe-reference[Probe schema reference].
 
 [id='con-common-configuration-prometheus-{context}']
-=== `metrics`
+=== `metricsConfig`
 
-Use the `metrics` property to enable and configure Prometheus metrics.
+Use the `metricsConfig` property to enable and configure Prometheus metrics.
 
-The `metrics` property can also contain additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
+The `metricsConfig` property contains a reference to the ConfigMap containing additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
 Strimzi supports Prometheus metrics using Prometheus JMX exporter to convert the JMX metrics supported by Apache Kafka and ZooKeeper to Prometheus metrics.
 
-To enable Prometheus metrics export without any further configuration, you can set it to an empty object (`{}`).
+To enable Prometheus metrics export without further configuration, you can reference a ConfigMap containing empty file under a specified key.
```

Comment:
We should be clear that without any further configuration, ALL metrics will be exported with no renaming as at the source.